_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy.cpp

```diff
@@ -2398,6 +2398,22 @@ void Legacy::ConfigureTask(
     task.StatusReportId = std::max<mfxU32>(1, m_prevTask.StatusReportId + 1);
     task.bForceSync = !!(task.InsertHeaders & INSERT_BPSEI);
 
+    if (task.FrameType & MFX_FRAMETYPE_I)
+    {
+        task.m_minQP = CO2.MinQPI;
+        task.m_maxQP = CO2.MaxQPI;
+    }
+    else if (task.FrameType & MFX_FRAMETYPE_P)
```

Comment:
Hi Hongbo,

Since this max/min QP is per sequence settings, do we still need P/B settings here? It may not work, right?

Thanks,
Thomas

Reply:
The sequence setting is enough.